besu/src/main/java/org/hyperledger/besu/cli/subcommands/RetestethSubCommand.java

```diff
@@ -57,6 +58,12 @@
       description = "The path to Besu data directory (default: ${DEFAULT-VALUE})")
   private final Path dataPath = DefaultCommandValues.getDefaultBesuDataPath(this);
 
+  @CommandLine.Option(
+      names = {"--enable-1559"},
```

Comment:
Should be `--Xenable-1559` - experimental/early access are `-X` in main besu (as is EIP-1559), and the pattern should be followed in other entry points.